scripts/debug/add_feed_tags.py

```diff
@@ -63,26 +63,22 @@
     {"onestop_id":"f-c283-jamestownsklallam","tags":{"managed_by": "mjc"}},
     {"onestop_id":"f-c2qf-mountainline","tags":{"gtfs_data_exchange": "mountain-line"}},
     {"onestop_id":"f-9rb-caravan~or~us","tags":{"feed_id": "caravan-or-us", "managed_by": "trillium", "gtfs_data_exchange": "caravan-airport-transportation"}},
-    {"onestop_id":"f-9rb-cascadespoint~or~us","tags":{"feed_id": "cascadespoint-or-us", "managed_by": "trillium", "gtfs_data_exchange": "cascades-point"}},
```

Comment:
No need to edit this file. (It's just left over from past data maintenance.)

Reply:
I thought it might be something like that, but took them out just to keep things consistent. 

New commit undoes this change.